README.md

```diff
@@ -1,17 +1,19 @@
-German Translation Pack for ExpressionEngine 5.x
-================================================
+German Translation Pack for ExpressionEngine 4.x and 5.x
```

Comment:
Does this language pack really work for both 4 and 5? We removed some keys from 4, are you keeping those around? If so, we'd probably rather have a separate branch for each major version.

Reply:
No, doesn't it. I did not think very well. What should be done so that the language pack for EE 4.3.6 can be downloaded without being affected by further work on the language pack for 5.x?